app/models/host.rb

```diff
@@ -1594,6 +1594,19 @@ def get_pct_cpu_metric_from_child_vm_performances(metric, capture_interval, time
     perf_hash
   end
 
+  # Display or hide certain charts
+  def cpu_mhz_available?
+    true
+  end
```

Comment:
this almost feels like a `supports :cpu_mhz`

I agree that the model is to not support it here and to override in the providers that actually do support this